example/pythonlib/basic/1-simple/build.mill

```diff
@@ -1,27 +1,59 @@
 package build
 import mill._, pythonlib._
 
-// Docs for PythonLib
 object foo extends PythonModule {
-  object bar extends PythonModule {
-    def pythonDeps = Seq("pandas==2.2.3", "numpy==2.1.3")
+  def pythonDeps = Seq("pandas==2.2.3", "numpy==2.1.3")
+
+  object test extends PythonTests {
```

Comment:
This should have an example of a third-party dependency being used in the test suite as well